packages/devtools_server/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/flutter/devtools
 
 environment:
-  sdk: '>=2.3.0 <3.0.0'
+  sdk: '>=2.6.0 <3.0.0'
```

Comment:
move this back to 2.3 unless we really really need extension methods for the server package.

Reply:
done